plugins/api.c

```diff
@@ -52,6 +52,8 @@
 #endif
 #endif
 
+#define QPP_MINIMUM_VERSION 2 // TODO move to header
```

Comment:
Could this go in plugin-qpp.h?

Reply:
I think moving this to plugins/plugin.h makes more sense